modules/custom_operations/examples/calculate_grid/export_model.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import argparse
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+from .calculate_grid import CalculateGrid
+
+
+class MyModel(nn.Module):
+    def __init__(self):
+        super(MyModel, self).__init__()
+        self.calculate_grid = CalculateGrid()
+
+    def forward(self, x):
+        return self.calculate_grid.apply(x)
+
+
+def export(num_points, max_grid_extent):
+    # Generate a list of unique positions and add a mantissa
+    np.random.seed(32)
+    torch.manual_seed(11)
+
+    inp_pos = np.random.randint(0, max_grid_extent, [num_points, 3])
+    inp_pos = torch.tensor(inp_pos) + torch.rand(inp_pos.shape, dtype=torch.float32) # [0, 1)
+
+    model = MyModel()
+    with torch.no_grad():
+        torch.onnx.export(model, (inp_pos), 'model.onnx',
+                          input_names=['input'],
+                          output_names=['output'],
+                          operator_export_type=torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK)
+
+    ref = model(inp_pos).detach().numpy()
+
+    # Pad values with espetial end line (-1, 0, 0) and zeros
```

Comment:
```suggestion
    # Pad values with espeсial end line (-1, 0, 0) and zeros
```

Reply:
@slyalin "special"? or `Pad with sentinel values (-1,0,0) and zeros`

Reply:
Your variant is better, agree -- I just fixed grammar in the original word :)